examples/lib/defaultHtmlStepUi.js

```diff
@@ -25,6 +25,9 @@ function DefaultHtmlStepUi(_sequencer, options) {
     if (step.options && step.options.description)
       step.description = step.options.description;
 
+      var stepName = step.name;
+      var packageLink = require(`../../src/modules/${stepName}/info.json`)['docs-link'];
```

Comment:
You can just do `step['docs-link']` here, no `require` required. (And also not allowed in browsers)